src/core/lib/iomgr/tcp_client_windows.cc

```diff
@@ -145,6 +145,8 @@ static int64_t tcp_connect(grpc_closure* on_done, grpc_endpoint** endpoint,
   grpc_error_handle error;
   async_connect* ac = NULL;
   absl::StatusOr<std::string> addr_uri;
+  int af;
```

Comment:
Can someone tell me how to enable UseEventEngineClient so that I can test the event engine client code path?

Reply:
iomgr and EventEngine both work now.